ports/atmel-samd/mpconfigport.mk

```diff
@@ -2,20 +2,17 @@
 # $(MPY-TOOL) needs to know what kind of longint to use (if any) to freeze long integers.
 # This should correspond to the MICROPY_LONGINT_IMPL definition in mpconfigport.h.
 
-ifdef LONGINT_IMPL_NONE
+ifeq ($(LONGINT_IMPL),NONE)
 MPY_TOOL_LONGINT_IMPL = -mlongint-impl=none
 endif
 
-ifdef LONGINT_IMPL_MPZ
+ifeq ($(LONGINT_IMPL),MPZ)
 MPY_TOOL_LONGINT_IMPL = -mlongint-impl=mpz
 endif
 
-ifdef LONGINT_IMPL_LONGLONG
+ifeq ($(LONGINT_IMPL),LONGONG)
```

Comment:
Typo here, LONGONG -> LONGLONG

Reply:
Thank you. LONGLONG takes more flash than MPZ, so I was just putting it in for completeness.